css/homepage.scss

```diff
@@ -31,6 +31,67 @@
  }
 }
 
+#description {
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  .icon {
+    line-height: 1;
+    vertical-align: middle;
+    font-size: 20em;
+    color: #400727;
+  }
+  p {
+    line-height: 1.5em;
+    color: #6f7c82
+  }
+  ul {
+    display: flex;
+    font-size: 1.1em;
+    padding-top: 1em;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    li {
+      position: relative;
+      display: list-item;
+      &:first-child {
+        margin-right: 1.8em;
+        &:after {
+          content: "";
+          position: absolute;
+          right: -17px;
+          top: calc(50% - 2px);
```

Comment:
How will that behave for the [5% of users](http://caniuse.com/#feat=calc <http://caniuse.com/#feat=calc>) (most probably much more effective visitors considering the audience here) who don’t have `calc` support?